velox/common/base/tests/GTestUtils.h

```diff
@@ -63,6 +63,16 @@
       << "Expected error message to contain '" << (_errorMessage)          \
       << "', but received '" << status.message() << "'."
 
+#define VELOX_ASSERT_ERROR_CODE(_expression, _type, _errorCode)               \
```

Comment:
nit: perhaps, VELOX_ASSERT_RUNTIME_THROW_CODE(_expression, _errorCode) to match VELOX_ASSERT_RUNTIME_THROW

Reply:
Added, thanks.